multidict/_multidict.c

```diff
@@ -922,7 +923,7 @@ static PyMethodDef multidict_methods[] = {
     {
         "__class_getitem__",
         multidict_class_getitem,
-        METH_VARARGS | METH_STATIC,
+        METH_VARARGS | METH_CLASS,
```

Comment:
It is not so necessary but mb `METH_O | METH_CLASS`?

Reply:
Good idea, done.